package.json

```diff
@@ -39,7 +39,8 @@
     "webpack": "^2.2.1"
   },
   "dependencies": {
-    "math-nodes": "^0.1.2",
-    "math-traverse": "^0.2.1"
+    "jest-cli": "^20.0.4",
```

Comment:
`jest` is only used for testing so it should listed in `"devDependencies"` not `"dependencies"`.